app/src/components/widgets/multiLevelWidgets/componentHealthCheck/componentHealthCheck.jsx

```diff
@@ -302,6 +302,7 @@ export class ComponentHealthCheck extends Component {
     const { activeBreadcrumbs, activeBreadcrumbId, isLoading } = this.state;
     const groupItems = this.getGroupItems();
     const isClickableGroupItem =
+      this.getBreadcrumbs().length > 1 &&
```

Comment:
```
const breadcrumbs = this.getBreadcrumbs();
const isGroupItemClickable =
      breadcrumbs.length > 1 &&
      activeBreadcrumbId !== (activeBreadcrumbs && activeBreadcrumbs.length - 1);
```
Also please change `this.getBreadcrumbs()` to `breadcrumbs ` at [this](https://github.com/reportportal/service-ui/pull/2017/files#diff-31e8925908c1cb64b03aabb3b99dbd9dR311) line